ajax/libs/quasar-framework/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "quasar-framework",
+  "filename": "quasar.mat.umd.min.js",
+  "version": "0.15.10",
+  "description": "Simultaneously build desktop/mobile SPA websites & phone/tablet apps with VueJS",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/quasarframework/quasar.git"
+  },
+  "keywords": [
+    "vuejs",
+    "vue",
+    "quasar",
+    "js",
+    "phone",
+    "tablet",
+    "desktop",
+    "spa",
+    "website",
+    "electron"
```

Comment:
Maybe we can add more keywords from repo topic, what do you think?
Like ios, android, electron, material, vue-components, material-theme, material-design, material-components ...
Thanks